### PR TITLE
FT0: first version of workflow, raw->digit convertation and QC

### PR DESCRIPTION
**Pull request by afurs**

### CODE REVIEW

**Comment by afurs** [comment]
Hello @teo ,
could you take a look to this PR please?

**Comment by teo** [comment]
Looks ok but I kindly ask @knopers8 to confirm

**Comment by afurs** [comment]
@teo thanks!
BTW, looks like checks are failed. Is it ok?

File: scripts/etc/ft0-digits-qc.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          "taskParameters": {
+            "ChannelIDs": "204,205"
+          },
+          "location": "local",
+          "localMachines": [
+            "localhost"
+          ],
+          "remoteMachine": "localhost",
+          "remotePort": "30132",
+          "mergingMode": "entire"
```
Comment (knopers8) [inline scripts/etc/ft0-digits-qc.json:46]: the workflow you generated does not use mergers, so this is not needed (but it does not hurt). out of curiosity, why did you put this? as far as i remember FT0 uses one FLP, so there is no need for merging anyway.

**Review by knopers8 — APPROVED** [review]
looks good to me

**Comment by knopers8** [comment]
One thing i forgot: can you add the script to the list in `scripts/generate-all-dpl-workflows.sh`? this way we can regenerate all the templates if something in DPL changes.

i noticed your hack for the QC cycle length configuration, this a good catch, i wasn't aware we can't modify it when running with ECS. I will note it down in jira.

File: scripts/ft0-digits-qc.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -x; # debug mode
```
Comment (knopers8) [inline scripts/ft0-digits-qc.sh:2]: and please comment the debug mode

**Comment by afurs** [comment]
Hello @knopers8 ,

I've made some modifications in FT0's dpl-workflow(some options were removed) in recent commit, also I commented debug mode line in script.

**Review by knopers8 — APPROVED** [review]
thanks

File: scripts/etc/ft0-digits-qc.json (hunk 1 above)
Comment (afurs) [reply scripts/etc/ft0-digits-qc.json:46]: > the workflow you generated does not use mergers, so this is not needed (but it does not hurt). out of curiosity, why did you put this? as far as i remember FT0 uses one FLP, so there is no need for merging anyway.

I was making some tests with this QC config, and forgot to remove "merging" option.

**Pull request merged**